ci_framework/roles/edpm_deploy/tasks/main.yml

```diff
@@ -68,6 +68,25 @@
       retries: "{{ cifmw_edpm_deploy_retries }}"
       delay: 20
 
+- name: Run gmake command manually
+  environment:
+    PATH: "{{ cifmw_path }}"
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+  ansible.builtin.shell: |
```

Comment:
IMHO it would be better to get a patch in install_yamls allowing to parametrize the `sed` thing, allowing then to run the ci_make